setup.py

```diff
@@ -14,10 +13,12 @@
 from setuptools.command.build_py import build_py
 from setuptools.command.develop import develop
 
-P2PD_VERSION = "v0.3.6"
-P2PD_CHECKSUM = "627d0c3b475a29331fdfd1667e828f6d"
-LIBP2P_TAR_URL = f"https://github.com/learning-at-home/go-libp2p-daemon/archive/refs/tags/{P2PD_VERSION}.tar.gz"
-P2PD_BINARY_URL = f"https://github.com/learning-at-home/go-libp2p-daemon/releases/download/{P2PD_VERSION}/p2pd"
+P2PD_VERSION = "v0.3.0-hivemind.8"
```

Comment:
Is there a reason for the `.8` suffix?

Reply:
Initially, we've forked `v0.3.0` of the upstream and incremented the last number for each batch of hivemind-specific modifications. I'm not sure it's okay, since if the upstream ever updates to `v0.3.1` and further, it will collude with the versions denoting our modifications. Thus, I suggest to switch to the new versioning scheme with such "post-releases" (e.g., `v0.3.0-hivemind.8` instead of `v0.3.8`).

Reply:
[reverted after discussion with @mryab]